openage/convert/service/export/png/binpack.pxd

```diff
@@ -35,6 +39,7 @@ cdef class BinaryTreePacker(Packer):
     cdef packer_node *root
 
     cdef void fit(self, block)
```

Comment:
`block` could be typed, either as a 3-tuple (which you use now) or using the struct type if you use a struct like I suggested.

Reply:
3-tuple would then be `cdef void fit(self, (unsigned int, unsigned int, unsigned int) block)`